src/Uno.UI/FeatureConfiguration.cs

```diff
@@ -265,6 +265,15 @@ public static class Popup
 			/// </summary>
 			public static bool UseNativePopup { get; set; } = true;
 #endif
+
+			/// <summary>
+			/// By default, light dismiss is disabled in UWP/WinUI unless
+			/// IsLightDismissEnabled is explicitly set to true.
+			/// In earlier versions of Uno Platform this property defaulted
+			/// to true, which was incorrect behavior. If your code depends on this
+			/// legacy behavior, use this property to override it.
+			/// </summary>
+			public static bool EnableLightDismissByDefault { get; set; } = false;
```

Comment:
Consider adding a `<remarks>` section indicating that AutoSuggestBox won't work correctly if the property is overridden to true.

Reply:
Yes, that's the case 👍